people/zuzana.json

```diff
@@ -1,21 +1,12 @@
 {
-
   "key": "zuzana",
-
   "name": "BcA. Zuzana Červínová",
-
   "nick": "Zuzana",
-
   "email": "zuzana.cervinova@mild.blue",
-
-  "title_cs": "JUNIOR FRONTEND DEVELOPER",
-
-  "title_en": "JUNIOR FRONTEND DEVELOPER",
-
+  "title_cs": "Juniorní frontend vývojářka",
```

Comment:
Podle me se rika Junior frontend vývojářka i v češtině. Ale hlavně tady jde o to, že v AJ se ty tituly nepoužívají. V ČJ tam mám Ing., ale v AJ necháváme jenom doktory. Nasťa to nějak schovává a podle mě tam je enum, který v sobě nemá BcA. A to je potřeba opravit. Nebo se pletu?

Reply:
titul jsem přidala do mild.blue a vytvořila nový pull request, kterému dělá review nasťa